src/TRestAxionOpticsMirror.cxx

```diff
@@ -322,8 +322,9 @@ Int_t TRestAxionOpticsMirror::ExportTables() {
     string path = REST_USER_PATH + "/export/";
 
     if (!TRestTools::fileExists(path)) {
-        cout << "Creating path: " << path << endl;
-        system(("mkdir -p " + path).c_str());
+        std::cout << "Creating path: " << path << std::endl;
+        int z = system(("mkdir -p " + path).c_str());
```

Comment:
We should use the <filesystem> api in my opinion (or some rest function built over it).

Reply:
So, please, commit the change